crates/db_schema_file/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "lemmy_db_schema_file"
+version.workspace = true
+edition.workspace = true
+description.workspace = true
+license.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+repository.workspace = true
+
+[lib]
+name = "lemmy_db_schema_file"
+path = "src/lib.rs"
+doctest = false
+
+[lints]
+workspace = true
+
+[features]
+full = ["diesel", "diesel_ltree", "diesel-derive-enum", "ts-rs"]
+
+[dependencies]
+serde = { workspace = true }
+strum = { workspace = true }
+diesel = { workspace = true, optional = true }
+diesel_ltree = { workspace = true, optional = true }
+ts-rs = { workspace = true, optional = true }
+diesel-derive-enum = { workspace = true, optional = true }
```

Comment:
Here are the moved files. `lemmy_db_schema_file` is a bit awkward name, but it avoids moving dozens or hundreds of files in `lemmy_db_schema` to a new location. Enums also had to be moved here because they need to be in the same crate as schema.